### DISCUSSION TITLE
Getting error during Skywalking installation in AKS

### DISCUSSION THREAD

**Discussion by parthasarathicentific** [apache/skywalking#12132] [Q&A] [answered]
Hi Team,

Getting error during Skywalking installation in AKS.  Need your help to resolve the problem.

I am running following command to install Apache Skywalking. But it is giving me below mentioned error. How to resolve it.

**Commands:**
_helm repo add skywalking https://apache.jfrog.io/artifactory/skywalking-helm
helm repo update_

_helm install skywalking skywalking/skywalking \
    --set oap.storageType=postgresql \
    --set oap.storage.postgresql.host=<my dbserver ip> \
    --set oap.storage.postgresql.port=5432 \
    --set oap.storage.postgresql.database=<my db name> \
    --set oap.storage.postgresql.user=<my db user>\
    --set oap.storage.postgresql.password=<my db user password> \
    --set oap.image.tag=9.6.0 \
    --set ui.image.tag=9.6.0 \
    --set oap.service.type=LoadBalancer \
    --set oap.service.annotations."service.beta.kubernetes.io/azure-load-balancer-internal"="true" \
    --set ui.service.type=LoadBalancer \
    --set ui.service.annotations."service.beta.kubernetes.io/azure-load-balancer-internal"="true" \
    --namespace skywalking_

**Error message:**

_Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: resource mapping not found for name: "elasticsearch-master-pdb" namespace: "" from "": no matches for kind "PodDisruptionBudget" in version "policy/v1beta1"
ensure CRDs are installed first_

Kindly help me to resolve the problem

**Comment by kezhenxu94** [comment, marked as answer]
If you are using other storage than elasticsearch, please disable elasticsearch (`--set elasticsearch.enabled=false`)

**Reply by parthasarathicentific** [reply]
Thanks for the reply.
